src/Playwright/Transport/Protocol/ArtifactInitializer.cs

```diff
@@ -0,0 +1,6 @@
+namespace Microsoft.Playwright.Transport.Protocol
+{
+    internal class ArtifactInitializer
```

Comment:
Do we need this for deserialization purposes? Any way we can avoid having an empty class?

Reply:
We don't need it :)

Reply:
removed